### ISSUE TITLE
Job container env was not properly set: PAI_CONTAINER_HOST_$type_PORT_LIST, PAI_TASK_ROLE_$name_HOST_LIST

### ISSUE THREAD

**Issue by hao1939** [Microsoft/pai#1895]
For the job containers, `PAI_CONTAINER_HOST_$type_PORT_LIST` were not properly set.
http://52.247.220.239/view.html?username=core&jobName=yuanya_multi_machine_test12-debug

For example the `client` role can't get the port list of:

> PAI_CONTAINER_HOST_tensorflow0_PORT_LIST
> PAI_CONTAINER_HOST_tensorflow1_PORT_LIST

Job config:
```
{
  "jobName": "yuanya_multi_machine_test12-debug",
  "image": "yann4leckie/nni:tf1.12-py3-hdfs",
  "authFile": "",
  "dataDir": "$PAI_DEFAULT_FS_URI/l1/bert/yuanya/mrpc",
  "outputDir": "$PAI_DEFAULT_FS_URI/l1/bert/yuanya/output",
  "codeDir": "$PAI_DEFAULT_FS_URI/l1/bert/yuanya/code",
  "virtualCluster": "default",
  "gpuType": "",
  "retryCount": 0,
  "taskRoles": [
    {
      "name": "woker0",
      "taskNumber": 1,
      "cpuNumber": 1,
      "memoryMB": 65434,
      "shmMB": 64,
      "gpuNumber": 0,
      "minFailedTaskCount": 1,
      "minSucceededTaskCount": null,
      "command": "env && sleep infinity",
      "portList": [
        {
          "label": "tensorflow0",
          "beginAt": 0,
          "portNumber": 1
        }
      ]
    },
    {
      "name": "woker1",
      "taskNumber": 1,
      "cpuNumber": 1,
      "memoryMB": 65434,
      "shmMB": 64,
      "gpuNumber": 0,
      "minFailedTaskCount": 1,
      "minSucceededTaskCount": null,
      "command": "env && sleep infinity",
      "portList": [
        {
          "label": "tensorflow1",
          "beginAt": 0,
          "portNumber": 1
        }
      ]
    },
    {
      "name": "client",
      "taskNumber": 1,
      "cpuNumber": 1,
      "memoryMB": 65434,
      "shmMB": 64,
      "gpuNumber": 0,
      "minFailedTaskCount": 1,
      "minSucceededTaskCount": null,
      "command": "env && sleep infinity",
      "portList": []
    }
  ]
}
```

**Comment by mzmssg** [comment]
It seems all right? I can see full list on all nodes. Below are client and worker0 output
![image](https://user-images.githubusercontent.com/11887940/50063817-fbcbf900-01e8-11e9-8f48-fe1cd111265f.png)
![image](https://user-images.githubusercontent.com/11887940/50063857-24ec8980-01e9-11e9-9437-ade0d60a7b0b.png)

**Comment by mzmssg** [comment]
For port list, every task can see its own ports, but can't get others port info

**Comment by wangcan0329** [comment]
Fixed in PR [#1918](https://github.com/Microsoft/pai/pull/1918).
